src/main/accumulMod.F90

```diff
@@ -711,7 +718,11 @@ subroutine accumulRest( ncid, flag )
 
     do nf = 1,naccflds
 
-       varname = trim(accum(nf)%name) // '_VALUE'
+       if (accum(nf)%old_name /= "") then
+          varname = trim(accum(nf)%name) // '_VALUE:' // trim(accum(nf)%old_name) // '_VALUE'
+       else
+          varname = trim(accum(nf)%name) // '_VALUE'
+       end if
```

Comment:
What you did for this backwards compatibility looks great - thanks! However:

- [ ]  I think you also need to do the same backwards compatibility thing for NSTEPS and PERIOD in accumulRest. These may not be needed in this case, but I think it will be important if people try to use this new old_name in other cases where NSTEPS/PERIOD are needed. However, let me know if you think otherwise based on your understanding of this module. (Side note: I'm not sure why PERIOD needs to be on the restart file in the first place, but let's not go down that rabbit hole.)

Reply:
Good point, done (22bda7d).